datafusion/sqllogictest/test_files/explain.slt

```diff
@@ -279,13 +279,97 @@ physical_plan
 GlobalLimitExec: skip=0, fetch=10, statistics=[Rows=Inexact(10), Bytes=Absent, [(Col[0]:),(Col[1]:),(Col[2]:)]]
 --CsvExec: file_groups={1 group: [[WORKSPACE_ROOT/datafusion/core/tests/data/example.csv]]}, projection=[a, b, c], limit=10, has_header=true, statistics=[Rows=Absent, Bytes=Absent, [(Col[0]:),(Col[1]:),(Col[2]:)]]
 
-# Parquet scan with statistics collected
+# Collect statistics
 statement ok
 set datafusion.execution.collect_statistics = true;
 
+# CSV scan
+# query having almost presentative opererators
+query TT
+explain
+SELECT t1.c1, sum(t2.c4)
+FROM aggregate_test_100_with_order as t1, aggregate_test_100_with_order as t2
+WHERE t1.c3 > 10 and t1.c11 != 30.5
+    AND t2.c13 = 'whatever'
+    AND t1.c2 = t2.c2 and t2.c10 < 987654321
+GROUP BY t1.c1
+HAVING sum(t2.c4) > 1
+ORDER BY  t1.c1 ASC
+LIMIT 10;
```

Comment:
@alamb : At first I thought I would add many small queries and on single data type but it turns out, if I do so, I have to add quite many and also many combinations for different data types. Still, they do not cover the propagation I want to see. Thus, I have to add complicated queries plus those small queries.

To avoid that, I decided to go with 2 quite representative queries, one on CVS file and one on parquet file. Each I have different combinations of filters on different data types and includes common standard SQL clauses (select, from, where, group by, having, order by, limit). They not only show the statistics for each operator but also how they are propagated upward.

Let me know what you think. I am happy to add small queries, too

Reply:
I think this makes sense and is a great first step.